README.md

```diff
@@ -102,6 +102,16 @@ var bre = new RulesEngine.RulesEngine(wfr, null);
 
 *Note: For each level of nested rules expected, a ThenInclude query appended will be needed as shown above.*
 
+### Editor
```

Comment:
Hi @alexreich, thanks for building this.
Unfortunately we can't promote it directly as it is not an official Editor.

I would recommend moving it below `How it works` section under the heading of `3rd party tools`